.github/workflows/plugin-check.yml

```diff
@@ -0,0 +1,39 @@
+name: WordPress Plugin Checks
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    uses: 10up/classifai/.github/workflows/build-release-zip.yml@feature/plugin-check-action # TODO: Update branch before merge.
+
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Download built zip
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ github.event.repository.name }}
+          path: ${{ github.event.repository.name }}
+
+      - name: Display structure of downloaded files
+        run: ls -R
+        working-directory: ${{ github.event.repository.name }}
+
+      - name: Run plugin check
+        uses: wordpress/plugin-check-action@v1
```

Comment:
Are there any options to know specifically what checks we're running or is it all-or-nothing?

Reply:
By default it will run all tests but you can exclude certain checks or certain categories of checks. If you see a few lines below, we're using the `exclude-checks` setting to remove a couple checks but we run everything else.

You can see all the checks and categories here: https://github.com/wordpress/plugin-check-action/?tab=readme-ov-file#supported-checks